### DISCUSSION TITLE
Pointer events have different timings

### DISCUSSION THREAD

**Discussion by seobits** [pixijs/pixijs#7681] [Q&A]
While trying to capture pointer events and then using the delta between the event fired, I found out that the deltas are different when you have the developer tools panel open, when the developer tools panel is closed it seems that the event are fired at the end of every frame.

This might not be an issue in PIXI itself but the browser doing something unintended, does anyone has more information about why is it behaving like this?

### Expected Behavior
Time between events should not be different when developer tools panel is open

### Current Behavior
Time between event is different when opening developer tools panel
Developer panel open: As soon as it is detected(average 1ms)
Developer panel closed: on the next frame after it has been detected(average 16ms@60fps)

### Steps to Reproduce
Access the link and move the mouse around on top of the canvas, it will have a different delta when the developer tools panel is open.

### Environment
- **`pixi.js` version**: 5.3.x
- **Browser & Version**: Chrome 92, Edge 92
- **OS & Version**:  Windows 10
- **Running Example**:  https://www.pixiplayground.com/#/edit/N1HC1jF_9lhKN7Za5UVEc

**Comment by bigtimebuddy** [comment]
Converting this to a discussion, since it seems largely to be a question about browser behavior.
